test/e2e/pages/allocation-details.js

```diff
@@ -0,0 +1,54 @@
+import { endOfWeek, format } from 'date-fns'
+import { Selector } from 'testcafe'
+
+import { fillInForm } from '../_helpers'
+
+import Page from './page'
+
+class AllocationDetailsPage extends Page {
+  constructor() {
+    super()
+    this.url = '/allocation/new/allocation-details'
+    this.nodes = {
```

Comment:
Could we also rename this to `this.fields` and `this.nodes` please. I think it would continue to upload the previous pattern and make it clear what the fields on the page are, compared to other elements.

Reply:
Ok